app/invocation/invocation_breakdown_card.tsx

```diff
@@ -6,6 +6,7 @@ import { getChartColor } from "../util/color";
 
 interface Props {
   durationMap: Map<string, number>;
+  durationByCategoryMap: Map<string, number>;
```

Comment:
seeing as these are both Map<string,number> with very similar names, it's probably worth documenting what the difference is and what can be found in each one

Reply:
Done.